src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -79,24 +79,16 @@ public Task InitLinter(IEnumerable<Rule> rules, CancellationToken cancellationTo
 
         public async Task<AnalysisResponse> AnalyzeJs(string filePath, CancellationToken cancellationToken)
         {
-            try
+            var analysisRequest = new AnalysisRequest
             {
-                var analysisRequest = new AnalysisRequest
-                {
-                    FilePath = filePath,
-                    IgnoreHeaderComments = true,
-                    TSConfigFilePaths = Array.Empty<string>() // eslint-bridge generates a default tsconfig for JS analysis
-                };
+                FilePath = filePath,
+                IgnoreHeaderComments = true,
+                TSConfigFilePaths = Array.Empty<string>() // eslint-bridge generates a default tsconfig for JS analysis
+            };
 
-                var responseString = await httpWrapper.PostAsync("analyze-js", analysisRequest, cancellationToken);
+            var responseString = await httpWrapper.PostAsync("analyze-js", analysisRequest, cancellationToken);
 
-                return responseString == null ? null : JsonConvert.DeserializeObject<AnalysisResponse>(responseString);
-            }
-            catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
```

Comment:
I decided to remove the try-catch here. I need an outer try-catch in the caller (JavaScriptAnalyzer) anyway and this one doesn't add anything.